pytest-backend/pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[tool.poetry]
+name = "pytest-exasol-backend"
+version = "0.1.0"
+description = ""
+authors = ["Mikhail Beck <mikhail.beck@exasol.com>"]
+readme = "README.md"
+packages = [{include = "exasol"}]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+pytest = ">=7,<9"
+exasol-saas-api = ">=0.6.0,<1.0.0"
+pytest-exasol-itde = { path = "../pytest-itde" }
+pytest-exasol-saas = { path = "../pytest-saas" }
+
+[tool.poetry.plugins.pytest11]
+backend = "exasol.pytest_backend"
```

Comment:
Maybe
```
exasol-connection = "exasol.pytest_exasol_connection"
```
or
```
exasol-backend = "exasol.pytest_exasol_backend"
```
?

Reply:
I think it's the same naming question as in the other comment. I don't mind. The intuition for the name was mainly the similarity to the other two plugins. Why do we need two "exasol" in the full name?